Lesson_14/Tusk_1.py

```diff
@@ -0,0 +1,115 @@
+import json
+import re
+
+print('Phone list program is started.')
+
+commands = ('You can use next commands:'
+            '\n  - stats: If you want to see the quantity of records'
+            '\n  - add: if you want to add a new record'
+            '\n  - delete <name>: if you want to delete a record with name in <>'
+            '\n  - list: to see the list of all records'
+            '\n  - show <name>: to see the information about the name in <>'
+            '\n  - commands: if you want to see tha command list again'
+            '\n  - exit: if you want to exit the program')
+print(commands)
+file_address = 'phone_book.txt'
+
+try:
+    with open(file_address, 'r') as file:
+        json_data = file.read()
+    phone_list = json.loads(json_data)
+except FileNotFoundError:
+    phone_list = {}
+
+while True:
+    command = input('Please enter a command: ').lower()
+
+    if command == 'stats':
+        if len(phone_list) == 0:
+            print('There is no records for now')
+        else:
+            print(len(phone_list))
+
+    elif command == 'add':
+        name = input('Please enter a name: ')
+        if name in phone_list:
+            print('This name is already here')
+            continue
+
+        while True:
```

Comment:
I don't think `while True` is a good choice here. Maybe it is better to skip this command and start from the beginning when a user enters the wrong format?